### ISSUE TITLE
metric depth estimation finetuned with nuscenes

### ISSUE THREAD

**Issue by xiaobh1519** [LiheYoung/Depth-Anything#172]
Hi, thanks for your great work!
I have tried a lot of experiments to fintune your metric depth estimation model with nuscenes dataset. But I find that the results turns worse ( “corrugations” appear) after about 3K steps like images below. The situation becomes more intense as  training steps increases.
![image](https://github.com/LiheYoung/Depth-Anything/assets/79265019/f610af23-5d0a-470e-8dcb-d00713444a67)
![image](https://github.com/LiheYoung/Depth-Anything/assets/79265019/5063042d-5045-4005-8a76-7120ada00c40)
![image](https://github.com/LiheYoung/Depth-Anything/assets/79265019/f8294655-c72f-4420-9648-a172654f7b0e)
And I found that the effect of this phenomenon varies for images with different input resolutions. Can you tell me why this phenomenon occurs?
I have been troubled by this issue for a long time. I would be extremely grateful if I could get your answer.

**Comment by XYQ0121** [comment]
@xiaobh1519 Hello, I am also fine-tuning metric depth estimation using a custom dataset. I plan to use the KITTI config. My image size is 2160*3840. Can you tell me what modifications you made?Thanks a lot!
![Snipaste_2024-05-15_10-12-04](https://github.com/LiheYoung/Depth-Anything/assets/110807624/02fcb0b5-14fc-40d5-b939-d7f50e5e3b81)

**Comment by XYQ0121** [comment]
Even though I changed the image size to 2160*3840 in the KITTI config, it does not match the printed values from the config. Is there any problem with this situation？
![Snipaste_2024-05-15_10-58-15](https://github.com/LiheYoung/Depth-Anything/assets/110807624/0f9c3a77-2df9-405f-9a7a-580e23e83fc5)

**Comment by LiheYoung** [comment]
Hi @xiaobh1519, did you try to decrease the learning rate of our pre-trained encoder?

**Comment by vvonjoon** [comment]
I have an exactly same issue as @xiaobh1519 . Has anyone solved this problem?
